### PR TITLE
Updating GPT notebooks to use mpt-7b

### PR DESCRIPTION
**Pull request by zredlined**
Added `calc_epochs` function to calculate epochs for an arbitrary dataset to complete training in < 30 minutes

### CODE REVIEW

File: config_templates/gretel/synthetics/natural-language.yml
Comment (kboyd) [inline config_templates/gretel/synthetics/natural-language.yml:11]: Do changes to yaml blueprints need to go through the dev branch before being merged to the main?

Comment (kboyd) [reply config_templates/gretel/synthetics/natural-language.yml:11]: Right, notebooks are handled differently, But you have changes to the config template and (now) use case cards in this PR. I guess I don't understand the develop -> main processes for this repo.

**Review by kboyd — APPROVED** [review]
Approving for model and content changes. Would prefer to get approval from @johntmyers for the process stuff with this repo before merging.

File: config_templates/gretel/synthetics/natural-language.yml
Comment (thesteady) [reply config_templates/gretel/synthetics/natural-language.yml:11]: Changes that go directly to `main` in this repo won't be reflected in development environments (e.g., Console localhost and console-eng). For changes that could break things in Console (such as reworking use cases) we like to put through develop first so we can review.

File: use_cases/gretel.json
Comment (thesteady) [inline use_cases/gretel.json:80]: you'll want to generate a sample data preview for this new CSV file. There's a script in this repo to do so.

The change to this file is one we really should update in develop first and then cherry-pick to main, so that we don't have future issues when we promote things from develop -> main in our normal Console development workflow. Sorry for any inconvenience.

Comment (yamini) [reply use_cases/gretel.json:80]: @thesteady yep, Alex and I just discussed this step. @zredlined can you remove this file from the PR? There's some hard coded metadata about the sample dataset in gretel.json as well. I can take care of updating gretel.json and running the preview generation script in a separate PR in `develop`.

Comment (yamini) [reply use_cases/gretel.json:80]: oh, i see you updated the metadata as well.

Comment (yamini) [reply use_cases/gretel.json:80]: Generated the dataset preview and pushed it to this branch.

**Review by thesteady — APPROVED** [review]
we'll pull the new stuff back into develop separately.

**Review by thesteady — APPROVED** [review]
we'll pull the new stuff back into develop separately.

**Pull request merged**
